src/generate_website.zig

```diff
@@ -74,47 +248,54 @@ pub fn main() !void {
         switch (result.kind) {
             .success => {
                 count = try std.fmt.allocPrint(arena,
-                    \\{} runs
+                    \\<strong style="color: var(--green);">{} runs</strong>
                 , .{
                     result.total_execs,
                 });
             },
             .crash => {
                 count = try std.fmt.allocPrint(arena,
-                    \\{} crashes
+                    \\<strong style="color: var(--magenta);">{} crashes</strong>
                 , .{
                     result.unique_crashes,
                 });
             },
             .hang => {
                 count = try std.fmt.allocPrint(arena,
-                    \\{} hangs
+                    \\<strong style="color: var(--orange);">{} hangs</strong>
                 , .{
                     result.unique_hangs,
                 });
             },
         }
+
+        const coverage_percent = result.edges_found * 100 / result.total_edges;
+        const coverage_display = try std.fmt.allocPrint(arena,
+            \\{}/{} ({d}%)
+        , .{
+            result.edges_found,
+            result.total_edges,
+            coverage_percent,
+        });
+
         try buffered.writer().print(
             \\  <tr>
             \\    <td><a href="https://github.com/roc-lang/roc/tree/{s}">{s}</a>{s}</td>
             \\    <td>{s}</td>
             \\    <td>{s}</td>
-            \\    <td>{}</td>
-            \\    <td>{}/{} ~ {}%</td>
+            \\    <td><time data-timestamp="{}"><!-- Will be replaced by javascript on load --></time></td>
+            \\    <td>{s}</td>
             \\    <td>{s}</td>
             \\  </tr>
             \\
         , .{
             result.commit_sha,
             result.commit_sha[0..7],
             if (branch) |b| b else "",
-            // TODO add link to fuzzer specific page.
             result.fuzzer,
-            if (cmd) |c| c else "No Failures!",
+            if (cmd) |c| c else "<i>Nil failures</i>",
```

Comment:
Why nil?

Reply:
Because thats the technical term.
 https://dictionary.cambridge.org/dictionary/english/nil

Reply:
I don't disagree. Just not terminology I would use generally for this.